docs/coordinates.rst

```diff
@@ -0,0 +1,444 @@
+       [(3600.        , 4.56905253e-02), (3600.        , 1.80000000e+03),
+        (3600.        , 3.59995431e+03), (3600.        , 5.39963453e+03)],
+       [(5039.92178073, 4.56894119e-02), (5040.03143627, 1.79995614e+03),
+        (5040.14110851, 3.59986658e+03), (5040.25079745, 5.39950295e+03)],
+       [(6479.70323031, 4.56860725e-02), (6479.92250932, 1.79982456e+03),
+        (6480.14182173, 3.59960344e+03), (6480.36116753, 5.39910830e+03)]]>)
+
+If the user wants the world coordinates for all the axes, ``axes`` can be set to ``None``, which is in fact the default.
+
+.. code-block:: python
+
+  >>> my_cube.axis_world_coords()
+  (<SpectralCoord [1.02e-09, 1.04e-09, 1.06e-09, 1.08e-09, 1.10e-09] m>, <SkyCoord (Helioprojective: obstime=None, rsun=695700.0 km, observer=None): (Tx, Ty) in arcsec
+      [[(2160.07821927, 4.56894119e-02), (2159.96856373, 1.79995614e+03),
+        (2159.85889149, 3.59986658e+03), (2159.74920255, 5.39950295e+03)],
+       [(3600.        , 4.56905253e-02), (3600.        , 1.80000000e+03),
+        (3600.        , 3.59995431e+03), (3600.        , 5.39963453e+03)],
+       [(5039.92178073, 4.56894119e-02), (5040.03143627, 1.79995614e+03),
+        (5040.14110851, 3.59986658e+03), (5040.25079745, 5.39950295e+03)],
+       [(6479.70323031, 4.56860725e-02), (6479.92250932, 1.79982456e+03),
+        (6480.14182173, 3.59960344e+03), (6480.36116753, 5.39910830e+03)]]>)
+
+By default `~ndcube.NDCube.axis_world_coords` returns the coordinates at the center of each pixel.
+However, the pixel edges can be obtained by setting the ``edges`` kwarg to ``True``.
+For example:
+
+.. code-block:: python
+
+  >>> my_cube.axis_world_coords(edges=True)
+  (<SpectralCoord [1.01e-09, 1.03e-09, 1.05e-09, 1.07e-09, 1.09e-09, 1.11e-09] m>, <SkyCoord (Helioprojective: obstime=None, rsun=695700.0 km, observer=None): (Tx, Ty) in arcsec
+      [[(1440.24341188, -899.79647591), (1440.07895112,  899.95636786),
+        (1439.91446531, 2699.84625127), (1439.74995445, 4499.59909505),
+        (1439.58541853, 6298.94094507)],
+       [(2880.05774973, -899.84032206), (2880.00292413,  900.00022848),
+        (2879.94809018, 2699.97783871), (2879.89324788, 4499.81838925),
+        (2879.83839723, 6299.24788597)],
+       [(4319.94225027, -899.84032206), (4319.99707587,  900.00022848),
+        (4320.05190982, 2699.97783871), (4320.10675212, 4499.81838925),
+        (4320.16160277, 6299.24788597)],
+       [(5759.75658812, -899.79647591), (5759.92104888,  899.95636786),
+        (5760.08553469, 2699.84625127), (5760.25004555, 4499.59909505),
+        (5760.41458147, 6298.94094507)],
+       [(7199.36047891, -899.70880283), (7199.63452676,  899.86866585),
+        (7199.90861634, 2699.58313412), (7200.18274766, 4499.1606028 ),
+        (7200.45692072, 6298.32719784)]]>)
+
+`~ndcube.NDCube.axis_world_coords` also allows the user to pick which WCS object should be used, `ndcube.NDCube.wcs` or `ndcube.NDCube.combined_wcs` by setting the ``wcs=`` keyword.
```

Comment:
you can also just pass `ndcube.extra_coords` to get the EC tables back.

Reply:
Currently that doesn't work.